doc/themes/the-things-stack/layouts/shortcodes/proto/term.html

```diff
@@ -0,0 +1 @@
+<a class="term" href="{{ printf "/glossary/#%s" (.Get 0) | relURL }}">{{ .Get 0 }}</a>
```

Comment:
Technically this doesn't belong in the theme, because it relies on a "glossary" that is located at "/glossary/". I think this belongs in `doc/layouts/shortcodes/glossary/term.html`